draft-btw-add-home.txt

```diff
@@ -988,26 +990,24 @@ Internet-Draft       Encrypted DNS in Home Networks        November 2020
 
    o  Locator: 2001:db8:1::1
 
-   The client requests "https://doh.example.com/.well-known/resinfo" to
-   retrieve the list of the URI Templates associated with this
-   discovered server.
+   The client will use DEER [I-D.pauly-add-deer] to discover the DoH
+   templates supported by the DNS server at the Locator address.  In
+   addition to the checks included in DEER, clients should verify the
+   ADN matches the hostname in the DEER-provided templates.  The IP
```

Comment:
Hmm do we really want the restriction that the ADN matches the hostname in the DEER-provided templates? The ADN was originally for verifying the certificate, not for verifying the the hostname of the server, and I think maybe it should still be used only for that purpose. Surely as long as the server(s) in the DEER-provided templates are offering certificates with hostnames that match the ADN, that's all that matters?

Reply:
Hmm, you make a fair point. The cert is really what matters. I've updated accordingly.